digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/model/RoadAddressLinkPartitioner.scala

```diff
@@ -8,7 +8,7 @@ object RoadAddressLinkPartitioner extends GraphPartitioner {
 
   def partition[T <: RoadAddressLinkLike](links: Seq[T]): Seq[Seq[T]] = {
     val linkGroups = links.groupBy { link => (
-      link.anomaly.value, link.roadNumber, link.roadPartNumber, link.trackCode,
+      link.anomaly.value.equals(Anomaly.None), link.anomaly.value.equals(Anomaly.Illogical), link.roadNumber, link.roadPartNumber, link.trackCode,
```

Comment:
As long as we're not using Illogical (and we're not sure if we ever will) we might just remove that comparison and be happy to check if there is any anomaly or none.